src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,13 @@ Do you want to proceed?</value>
   <data name="FontAndColors_BraceMatching" xml:space="preserve">
     <value>Brace Matching</value>
   </data>
+  <data name="GoToImplementationMessage" xml:space="preserve">
+    <value>Locating implementations...</value>
```

Comment:
would being more descriptive here be useful? like "locating implementations of symbol-name"...